tic_tac_toe/game.py

```diff
@@ -63,22 +101,45 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    winning_combinations = (
+        ((0, 0), (0, 1), (0, 2)),
+        ((1, 0), (1, 1), (1, 2)),
+        ((2, 0), (2, 1), (2, 2)),
+        ((0, 0), (1, 0), (2, 0)),
+        ((0, 1), (1, 1), (2, 1)),
+        ((0, 2), (1, 2), (2, 2)),
+        ((0, 0), (1, 1), (2, 2)),
+        ((0, 2), (1, 1), (2, 0))
+    )
+
+    if any(combination for combination in winning_combinations if _is_winning_combination(board, combination, player)):
```

Comment:
took me a while to understand your code because it's a bit 'run-on' even though it works. not sure whether there's a clearer way to write this while still keeping it to 1 line.

in case you'd like another version - we did something similar (albeit over 3 lines)
```
    for combination in winning_combinations:
        if _is_winning_combination(board, combination, player):
            return player
```